CONTRIBUTING.md

```diff
@@ -62,3 +62,27 @@ We use [SassDoc] to document Bourbon. [Annotations] should be ordered:
 
 [SassDoc]: http://sassdoc.com
 [Annotations]: http://sassdoc.com/annotations
+
+## Goverance
+
+We, the maintainers of Bourbon, would like you to know what to expect when
+interacting with this project's repo.
+
+As a user of Bourbon, you can expect care to be put into official releases. The
+maintainers cannot guarantee that the `master` branch will have the same API
+forever.
+
+As someone who is reviewing pull requests or commenting on issues, you are
+expected to abide by our [code of conduct].
+
+As a contributor submitting a pull request or opening an issue, you can expect
+an initial response from a maintainer within two weeks. The maintainers cannot
+guarantee that we will accept all feature requests, and may fix bugs in ways
+other than how the contributor suggests. You are expected to abide by our
+[code of conduct].
+
+As a maintainer, you can merge pull requests (even your own) at any time.
+Maintainers are encouraged to request code review from others at thoughtbot or
+any relevant peers. You are expected to abide by our [code of conduct].
+
+Only members of thoughtbot may be maintainers.
```

Comment:
I'm of the opinion of: let's document what we have, and then change it to be what we want.